.github/workflows/documentation.yml

```diff
@@ -0,0 +1,48 @@
+name: Documentation
+
+on:
+  push:
+    branches: main
+    tags: '*'
+  workflow_dispatch:  # Allow manually triggering the workflow
+
+concurrency:
+  group: >-
+    ${{ github.workflow }}-${{ github.ref_type }}-
```

Comment:
any reason the PR number is in there? this group seems to be written for actions mainly triggered by PRs, but this action is (rightfully) not triggered by PRs, so we should search for something more suitable. Imo doc building should not be possible in parallel at all, so the group needs to reflect that

Reply:
probably only `${{ github.workflow }}` should be used

Reply:
This was just taken as-is from Foundry Dev Tools. I'll adjust accordingly.